### PR TITLE
Add release notes for GPU frustum culling

### LINKED ISSUE #1307: Write release notes for PR #12889: Implement GPU frustum culling.
https://github.com/bevyengine/bevy/pull/12889 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `..\release-content\0.14\release-notes\12889_Implement_GPU_frustum_culling.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by jgayfer**
First time writing release notes!

This admittedly feels pretty bland, mostly due to the fact that the change isn't clearly motivated yet. I'm not that familiar with the subject matter, so I wasn't able to glean exactly what the benefit is, though I can only assume that there are performance implications depending on the use case.

I think I have the bones in place, but might need some help filling in the gaps!

Fixes #1307.

### CODE REVIEW

**Comment by jgayfer** [comment]
@pcwalton Wouldn't mind a hand word crafting the motivation behind this one, or helping me better understand who would be interested in giving this a shot :)

File: release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md [hunk 1]
```diff
@@ -1 +1,34 @@
-TODO
+<!-- GPU Frustum Culling: https://github.com/bevyengine/bevy/pull/12889-->
+
+Frustum culling can now optionally be performed on the GPU.
```
Comment (alice-i-cecile) [inline release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md:3]:
```suggestion
Bevy's rendering stack is often CPU-bound: by shifting more work onto the GPU, we can better balance the load and render more shiny things faster.
Frustum culling is an optimization technique that automatically hides objects that are outside of a camera's view (its frustum).
In Bevy 0.14, users can choose to have this work performed on the GPU, depending on the performance characteristics of their project.
```

File: release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+    Camera3dBundle::default(),
+    NoCpuCulling
+));
+```
+
+**Note:** `GpuCulling` does not imply `NoCpuCulling`. To only perform frustum
+culling on the GPU, you need to add both `GpuCulling` and `NoCpuCulling` to the
+camera.
+
+```rust
```
Comment (alice-i-cecile) [inline release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md:28]: I think we can condense this section substantially, by only including this snippet and then leaving a comment above each line as needed.

File: release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md (hunk 1 above)
Comment (jgayfer) [reply release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md]: Much nicer, thanks.

File: release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md (hunk 2 above)
Comment (jgayfer) [reply release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md:10]: Re-worked to just have one example with comments.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**